### ISSUE TITLE
Firefox redux-devtools not updating

### ISSUE THREAD

**Issue by sshquack** [reduxjs/redux-devtools#1008]
Versions:
- Firefox 96.0
- redux-devtools 3.0.3
- macOS 12
- Apple M1 chip

Congrats on the new v3 release. I updated to the latest redux-devtools 3.0.3 for Firefox and Chrome today. Everything works in Chrome as expected. In Firefox the redux-devtools refreshes only in the beginning and then it appears stuck. In v2 everything was working fine so this appeared only in v3. Please see recorded video.

https://user-images.githubusercontent.com/5678459/149205261-1dca1f1a-fc27-4a2f-9e4d-b4ebcb13da04.mp4

Please let me know if you need me something to debug this issue.

**Comment by microlabig** [comment]
I support sshquack. Also updated everything (more accurately updated automatically)

I display in a separate window devtools. Then I update the application (react hot reload), but the devtools is also updated and does not show anything. I have to go to devtools and press `<F5>`

OS: Ubuntu 20.04
Browser: Opera 82.0.4227.43
DevTools: v3.0.2

**Comment by Methuselah96** [comment]
Odd, all seems to be working fine on my end:

https://user-images.githubusercontent.com/693755/149255268-78575856-2004-47b0-a105-53c0de73527f.mp4

OS: Windows 11
Browser: Firefox 95.0.2 (64-bit)
DevTools: v3.0.3

Any ideas of what else I can try to reproduce? Looks like you're in a private window in your video. I don't seem to be able to access Redux DevTools in a private window, so I'm not sure if that could make a difference.

**Comment by sshquack** [comment]
@Methuselah96 Here is how to enable running the extension in private window
![Screenshot 000032 2@2x](https://user-images.githubusercontent.com/5678459/149258169-5e976154-bd05-4ed0-abcb-ec3cbfc03480.png)

**Comment by sshquack** [comment]
@Methuselah96 Oops! This was my mis-configuration. It is working now. Here is what I changed in the redux-devtools preferences
![Screenshot 000034@2x](https://user-images.githubusercontent.com/5678459/149258281-86a0a139-97c3-4621-851e-7a7062d3b2f3.png)

And here it is working as expected.

https://user-images.githubusercontent.com/5678459/149258474-3225829b-fc4c-437e-9e90-3433c3c7a16f.mp4



Apologies for the hassle. Closing this issue. Thanks for quickly investigating this @Methuselah96
